docs/en/developer/create-web-component/VUE.md

```diff
@@ -39,16 +39,29 @@ vue create {component-name} --default
 
 # navigate into newly created component folder
 cd {component-name}
+
+# add additional dependencies for legacy browser support
+npm install @babel/{cli,plugin-transform-runtime,preset-env}
```

Comment:
Should this command include `--save-dev`?

Reply:
Yep, good catch, updated.